s3pd/__init__.py

```diff
@@ -135,6 +135,10 @@ def s3pd(url, processes=8, chunksize=67108864, destination=None, func=None,
     filesize = get_filesize(client, bucket, key)
     chunks = create_chunks(chunksize, filesize)
 
+    # Prevent multiprocessing children to fork
```

Comment:
This seems to be an indirect way of checking if the current process is a daemon or not. Why not just use:
```python
from multiprocessing import Pool, current_process

# [...]

if current_process().daemon == True:
    processes = 1
```